### DISCUSSION TITLE
Stylus gestures for certain tools?

### DISCUSSION THREAD

**Discussion by adamwfennell** [xournalpp/xournalpp#5811] [Ideas]
Hello,

I was wondering if there is a feature for stylus gestures. For example, scribbling over something erases it, circling it allows you to move it, etc.?

Thanks,
Adam

**Comment by rolandlo** [comment]
No, there is no such feature in Xournal++. Are you requesting one?

**Reply by adamwfennell** [reply]
I'd certainly request such a feature if it's not currently one!
